### PR TITLE
use product specific boot timeout proposal (jsc#SLE-22667)

### PR DESCRIPTION
**Pull request by jreidinger**
jira:  https://jira.suse.com/browse/SLE-22667

related: https://github.com/yast/yast-yast2/pull/1240 and https://github.com/yast/yast-installation-control/pull/122

### CODE REVIEW

**Comment by jreidinger** [comment]
tested manually and works as expected

File: src/lib/bootloader/grub2base.rb [hunk 1]
```diff
@@ -361,7 +361,7 @@ def propose_terminal
     def propose_timeout
       return if grub_default.timeout
 
-      grub_default.timeout = "8"
+      grub_default.timeout = Yast::ProductFeatures.GetIntegerFeature("globals", "boot_timeout").to_s
```
Comment (teclator) [inline src/lib/bootloader/grub2base.rb:364]: So, what is default if not defined?

Comment (jreidinger) [reply src/lib/bootloader/grub2base.rb:364]: It is always defined. See - https://github.com/yast/yast-yast2/pull/1240/files#diff-e3d91269dec42511487bf9598e632fe2da743b05435e0cf3c77da23e520c0f77R55

File: package/yast2-bootloader.changes [hunk 2]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Jan 28 22:37:00 UTC 2022 - Josef Reidinger <jreidinger@suse.com>
+
+- Allow product/system role to select boot timeout (jsc#SLE-22667)
+- 4.4.16
```
Comment (teclator) [inline package/yast2-bootloader.changes:5]: Just miss the version bump in the spec file.

Comment (jreidinger) [reply package/yast2-bootloader.changes:5]: good catch

File: src/lib/bootloader/grub2base.rb (hunk 1 above)
Comment (teclator) [reply src/lib/bootloader/grub2base.rb:364]: .. so, maybe the dependency should be updated requiring that specific version? https://github.com/yast/yast-bootloader/blob/0e1d4068e86aaf04a602a62ee30a4a72395aa1c2/package/yast2-bootloader.spec#L46

Comment (teclator) [reply src/lib/bootloader/grub2base.rb:364]: https://github.com/yast/yast-bootloader/blob/0e1d4068e86aaf04a602a62ee30a4a72395aa1c2/package/yast2-bootloader.spec#L29

**Review by teclator — APPROVED** [review]
LGTM

**Pull request merged**